src/input/plugins/CdioParanoiaInputPlugin.cxx

```diff
@@ -251,14 +254,25 @@ input_cdio_open(const char *uri,
 				      be);
 	}
 
-	lsn_t lsn_from, lsn_to;
-	if (parsed_uri.track >= 0) {
-		lsn_from = cdio_get_track_lsn(cdio, parsed_uri.track);
-		lsn_to = cdio_get_track_last_lsn(cdio, parsed_uri.track);
-	} else {
-		lsn_from = 0;
-		lsn_to = cdio_get_disc_last_lsn(cdio);
-	}
+	lsn_t lsn_to, lsn_from;
+
+	if(parsed_uri.track < 0){
```

Comment:
This reorders the if/else branches. Why?
Also you change the coding style by removing the space between `if(` - why?
Both just adds noise and makes your diff hard to review.

Reply:
Your right! I've missed to clean this up after testing. Sorry! Fixed!